mk/dotnet-packages-build.sh

```diff
@@ -141,6 +141,7 @@ run_msbuild_nofw()
 }
 
 cd ${SCRATCH_DIR}/xml-rpc.net/src && run_msbuild
+cd ${SCRATCH_DIR}/xml-rpc_v45.net/src && run_msbuild && mv ../bin/CookComputing.XmlRpcV2.dll ../bin/CookComputing.XmlRpcV2_dotnet45.dll && mv ../bin/CookComputing.XmlRpcV2.pdb ../bin/CookComputing.XmlRpcV2_dotnet45.pdb #building for dotnet4.5
```

Comment:
Oops, here's something I missed the first time I looked at this PR: here you need a different run_msbuild targeting .NET 4.5.